vars/tagDeployment.groovy

```diff
@@ -1,12 +1,19 @@
 #!/usr/bin/env groovy
 
-def call(String microservice, String aws_profile = "test", String tag = null) {
-  tag = tag ?: gitCommit()
-
-  build job: 'run-tag-and-capture-notes-commit-based',
-    parameters: [
-      string(name: 'ENVIRONMENT', value: aws_profile),
-      string(name: 'COMMIT_HASH', value: tag),
-      string(name: 'SERVICE_TO_TAG', value: microservice)
-    ]
+def call() {
+  commit_hash = gitCommit()
+
+  date = new java.text.SimpleDateFormat("yyyy-MM-dd-HH:mm:ss").format(new Date())
+
+  latest_tag_components = sh "git describe --abbrev=0 --match 'alpha_release-*'".returnStdout().trim().split('-')
```

Comment:
probably better to follow https://github.com/alphagov/pay-jenkins-library/blob/master/vars/tagPact.groovy#L10-L13, so 
```
sh (
  script: git describe --abbrev=0 --match 'alpha_release-*'
  returnStdout: true
).trim().split('-')
```